### PR TITLE
Add Sumo Logic documentation.

### PR DESCRIPTION
**Pull request by KyleTryon**
# Description
Add documentation and screenshots for installing and utilizing Sumo Logic Orb.

Preview: https://i.imgur.com/Uv7pv3I.png

### CODE REVIEW

File: jekyll/_cci2/insights.md [hunk 1]
```diff
@@ -35,10 +35,96 @@ The per-project insights page gives you access to the build status and build per
...
+          requires:
+            - build
+      - deploy:
+          requires:
+            - test
+```
+#### 3. Create two source collectors.
+You will need to create two "source collectors" on Sumo Logic which will return an HTTPS URL which will be used to send your job data to.
+
+You will need to create one called `circleci/job-collector` and another called `circleciworkflow-collector`.
```
Comment (KunalJain) [inline jekyll/_cci2/insights.md]: minor typo -> `circleci/workflow-collector`

File: jekyll/_cci2/insights.md [hunk 2]
```diff
@@ -35,10 +35,96 @@ The per-project insights page gives you access to the build status and build per
...
 Refer to the [Collecting Test Metadata]({{ site.baseurl }}/2.0/collect-test-data/) document for instructions to configure insights into your most failed tests.
+
+## Sumo Logic Integration
+
+Sumo Logic users may now track and visualize analytical data  across all of their jobs on CircleCI by utilizing the new Sumo Logic Orb and CircleCI App for Sumo Logic app integration on Sumo Logic's partner integrations site.
+
+
+### The CircleCI Dashboard for Sumo Logic
+
+![header]({{ site.baseurl }}/assets/img/docs/CircleCI_SumoLogic_Dashboard.png)
```
Comment (michelle-luna) [inline jekyll/_cci2/insights.md]: this needs a description for 508 compliance (images must have descriptive alt text).

File: jekyll/_cci2/insights.md [hunk 3]
```diff
@@ -35,10 +35,96 @@ The per-project insights page gives you access to the build status and build per
...
+6. Save the resulting URL.
+
+#### 4. Add environment variables.
+For each of the URLs produce in the previous step, create the corrisponding environment variable.
+
+Env vars:
+- `JOB_HTTP_SOURCE`
+- `WORKFLOW_HTTP_SOURCE`
+
+**[How to add an environment variable to your project.]({{ site.baseurl }}/env-vars/#setting-an-environment-variable-in-a-project)**
```
Comment (michelle-luna) [inline jekyll/_cci2/insights.md]:
```suggestion
**[How to add an environment variable to your project.]({{ site.baseurl }}2.0/env-vars/#setting-an-environment-variable-in-a-project)**
```

File: jekyll/_cci2/insights.md [hunk 4]
```diff
@@ -35,10 +35,96 @@ The per-project insights page gives you access to the build status and build per
...
+
+This will link the orb with your Sumo Logic dashboard.
+
+That's all! Your Sumo Logic dashboard will now begin to populate with data as each job runs on CircleCI.
```
Comment (michelle-luna) [inline jekyll/_cci2/insights.md]:
```suggestion
Your Sumo Logic dashboard will now begin to populate with data as each job runs on CircleCI.
```

File: jekyll/_cci2/insights.md [hunk 5]
```diff
@@ -35,10 +35,96 @@ The per-project insights page gives you access to the build status and build per
...
+To create the two source collectors:
+1. From the dashboard select the **Setup Wizard**.
+2. Select **Set Up Streaming Data**.
+3. Scroll to the bottom and select **All Other Sources**.
+4. Select **HTTPS Source**
+5. For the `Source Category` enter one of the two mentioned above.
+6. Save the resulting URL.
+
+#### 4. Add environment variables.
+For each of the URLs produce in the previous step, create the corrisponding environment variable.
```
Comment (michelle-luna) [inline jekyll/_cci2/insights.md]:
```suggestion
For each of the URLs produce in the previous step, create the corresponding environment variable.
```

File: jekyll/_cci2/insights.md [hunk 6]
```diff
@@ -35,10 +35,96 @@ The per-project insights page gives you access to the build status and build per
...
+      - build
+      - test:
+          requires:
+            - build
+      - deploy:
+          requires:
+            - test
+```
+#### 3. Create two source collectors.
+You will need to create two "source collectors" on Sumo Logic which will return an HTTPS URL which will be used to send your job data to.
```
Comment (michelle-luna) [inline jekyll/_cci2/insights.md]:
```suggestion
You will need to create two *source collectors* on Sumo Logic which will return an HTTPS URL. Your job data will be sent to this HTTPS URL.
```

File: jekyll/_cci2/insights.md [hunk 7]
```diff
@@ -35,10 +35,96 @@ The per-project insights page gives you access to the build status and build per
...
 
 - **Build Performance:** The Build Performance graph aggregates your build/job data for a particular day and plots the median for that day going back as far as 90 days. Monitor the performance of your repo by clicking a particular branch.
 
 ## See Also
 
 Refer to the [Collecting Test Metadata]({{ site.baseurl }}/2.0/collect-test-data/) document for instructions to configure insights into your most failed tests.
+
+## Sumo Logic Integration
+
+Sumo Logic users may now track and visualize analytical data  across all of their jobs on CircleCI by utilizing the new Sumo Logic Orb and CircleCI App for Sumo Logic app integration on Sumo Logic's partner integrations site.
```
Comment (michelle-luna) [inline jekyll/_cci2/insights.md]:
```suggestion
Sumo Logic users may track and visualize analytical data across all of their jobs on CircleCI. To do so, use the  Sumo Logic Orb and Sumo Logic app integration from the Sumo Logic partner integrations site.
```

File: jekyll/_cci2/insights.md [hunk 8]
```diff
@@ -35,10 +35,96 @@ The per-project insights page gives you access to the build status and build per
...
+- Total Successful Jobs
+- Total Failed Jobs
+- Job Outcome
+- Average Runtime in Seconds (by Job)
+- Jobs By Projects
+- Recent Jobs (latest 10)
+- Top 10 Slowest Failed Jobs In Seconds
+- Top 10 Slowest Successful Jobs In Seconds
+
+You can install the CircleCI dashboard via the App Catalog from the dashboard home page.
```
Comment (michelle-luna) [inline jekyll/_cci2/insights.md]:
```suggestion
Install the CircleCI dashboard by using the App Catalog from the dashboard home page.
```

File: jekyll/_cci2/insights.md [hunk 9]
```diff
@@ -35,10 +35,96 @@ The per-project insights page gives you access to the build status and build per
...
+
+![header]({{ site.baseurl }}/assets/img/docs/sumologic_app_catalog.png)
+
+This dashboard receives data via the CircleCI Sumo Logic orb which must be included in your projects to be tracked.
```
Comment (michelle-luna) [inline jekyll/_cci2/insights.md]:
```suggestion
This dashboard receives data through the CircleCI Sumo Logic orb which must be included in your projects to be tracked.
```

File: jekyll/_cci2/insights.md [hunk 10]
```diff
@@ -35,10 +35,96 @@ The per-project insights page gives you access to the build status and build per
...
+
+### The Sumo Logic Orb
+
+You can find the latest version of the Sumo Logic orb on the [Orb Registry](https://circleci.com/orbs/registry/orb/circleci/sumologic).
```
Comment (michelle-luna) [inline jekyll/_cci2/insights.md]:
```suggestion
Find the latest version of the Sumo Logic orb on the [Orb Registry](https://circleci.com/orbs/registry/orb/circleci/sumologic).
```

File: jekyll/_cci2/insights.md [hunk 11]
```diff
@@ -35,10 +35,96 @@ The per-project insights page gives you access to the build status and build per
...
+
+#### 1. Import the Sumo Logic orb.
+Add the Sumo Logic orb to your project by including the top-level "orbs" key and import `circleci/sumologic@x.y.z` as `sumologic`.
```
Comment (michelle-luna) [inline jekyll/_cci2/insights.md]:
```suggestion
Add the Sumo Logic orb to your project by including the top-level `orbs` key and import `circleci/sumologic@x.y.z` as follows, replacing `x.y.z` with the latest version number at the link above.
```

File: jekyll/_cci2/insights.md [hunk 12]
```diff
@@ -35,10 +35,96 @@ The per-project insights page gives you access to the build status and build per
...
+
+```yaml
+orbs:
+  sumologic: circleci/sumologic@x.y.z
+```
+
+#### 2. Add _Workflow-Collector_ to Workflow.
+The `workflow-collector` is a job that will run in parallel along side your workflow and send anyalytics to Sumo Logic until all of the jobs in your Workflow have complete.
```
Comment (michelle-luna) [inline jekyll/_cci2/insights.md]:
```suggestion
The `workflow-collector` is a job that will run in parallel along side your workflow and send anyalytics to Sumo Logic until all of the jobs in your workflow have completed.
```

**Comment by KyleTryon** [comment]
All suggestions and changes merged. @michelle-luna

**Pull request merged**